lms/djangoapps/discussion_api/api.py

```diff
@@ -494,12 +494,6 @@ def _check_editable_fields(cc_content, data, context):
     1. the given update data contains a field that is not editable by the requesting user.
     2. data contains other editable fields along with 'read' field.
     """
-    # updating 'read' field should not update activity date for thread
-    # while updating any other field should update activity date;
-    # hence, 'read' is mutually exclusive to all other editable fields.
-    if 'read' in data and len(data) > 1:
-        raise ValidationError("No Other fields should be updated with 'read' field.")
-
```

Comment:
@jcdyer The first implementation i did was to mark thread as read for user as part of cs_comment_service thread PUT endpoint; and that required to make it mutually exclusive so activity does not get affected.
But with the new commit I have changed the implementation and handled 'read' field as many other actionable items (such as 'abused_flag', 'vote', 'following'). We consider them as separate action in addition to thread update and hence does not effect recent activity at all. So yes, no need to make it mutually exclusive this way.